### PR TITLE
BugFix - Character Selection

### PR DESCRIPTION
**Pull request by m4rtint**
- Character selection bug fix - used controller dead zone. 
 - joystick press right 

- destructible OnDestroy unsubscribes instead of ondisable - ( strange behaviour)

### CODE REVIEW

**Comment by m4rtint** [comment]
[Controller doesnt work properly on player selection](https://trello.com/c/4pZmTuqB/719-controller-doesnt-work-properly-on-player-selection)

File: Assets/_PartyParkade/Script/Game/Destructables/Destructible.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ private void Start()
             StateManager.Instance.OnPreGame += CleanUp;
         }
 
-        private void OnDisable()
+        private void OnDestroy()
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/Game/Destructables/Destructible.cs:74]: So this was the reason the weird behaviour @wbeltran777 reported was happening?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/Destructables/Destructible.cs:74]: yuh

Comment (m4rtint) [reply Assets/_PartyParkade/Script/Game/Destructables/Destructible.cs:74]: Maybe OnDisable isn't called on scene change, but OnDestroy is called

**Pull request merged**